test/integration/vorpal-engine_test.go

```diff
@@ -0,0 +1,44 @@
+//go:build integration
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/checkmarx/ast-cli/internal/commands"
+	errorConstants "github.com/checkmarx/ast-cli/internal/constants/errors"
+	commonParams "github.com/checkmarx/ast-cli/internal/params"
+	"gotest.tools/assert"
+)
+
+func TestScanVorpal_NoFileSourceSent_FailCommandWithError(t *testing.T) {
```

Comment:
Add a happy case

Reply:
This is the happy case:
TestExecuteVorpalScan_CorrectFlagsSent_SuccessfullyReturnMockData
(Still need to add real file as we discussed in a call)